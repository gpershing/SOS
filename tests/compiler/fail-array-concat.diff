diff: ./tests/compiler/tests/compiler/fail-array-concat.err: No such file or directory
