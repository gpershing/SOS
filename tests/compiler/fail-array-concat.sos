array int x = [1, 2]
array float y = [1.1, 2.2]

array int z = x@y
