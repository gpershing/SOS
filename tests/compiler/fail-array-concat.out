Cannot concatenate arrays of different types
